linode/manager
# feat: [M3 7015] - Create nodebalancer tab in firewalls landing

## Description 📝
Added NodeBalancers in the Firewalls landing page.

## Major Changes 🔄
- Added new NodeBalancer tab
- Added the total number of each product as an integer next to the tab label

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-08-24 at 1 48 55 PM](https://github.com/linode/manager/assets/139489745/eb2918b5-ed3f-4ed3-89af-cab126d49cb1) | ![Screenshot 2023-08-24 at 1 49 06 PM](https://github.com/linode/manager/assets/139489745/fa2a18e9-524e-46af-9597-f909e39d6360) |
|  | ![Screenshot 2023-08-24 at 1 50 10 PM](https://github.com/linode/manager/assets/139489745/ccb8f067-da18-4deb-abeb-370d6edb9a35) |

## How to test 🧪
1. Navigate to http://localhost:3000/firewalls
2. Click on any of your firewalls
3. Ensure there is an extra NodeBalancers tab
4. Ensure the number next to the Linodes tab accurately represents the number of assigned linodes
5. Ensure the Linode tab functionality still works
6. Ensure the breadcrumb now has "Assigned [Device]", device is either a Linode or NodeBalancer
